rust/lance/src/index/vector/ivf/io.rs

```diff
@@ -296,8 +290,8 @@ pub(super) async fn write_hnsw_quantization_index_partitions(
     let mut part_files = Vec::with_capacity(ivf.num_partitions());
     let mut aux_part_files = Vec::with_capacity(ivf.num_partitions());
     let tmp_part_dir = Path::from_filesystem_path(TempDir::new()?)?;
-    let mut tasks = Vec::with_capacity(ivf.num_partitions());
-    let sem = Arc::new(Semaphore::new(*HNSW_PARTITIONS_BUILD_PARALLEL));
+    let mut aux_ivf = IvfData::empty();
```

Comment:
is this the right name?  why `aux_ivf` contains `IvfData`?

Reply:
`aux_ivf` is for tracking the storage partitions